instructor.html

```diff
@@ -72,58 +73,292 @@ <h6 class="mdc-drawer__subtitle" id="userEmailDisplay"></h6>
           </div>
         </aside>
         <div id="content-wrapper">
-          <div class="container-fluid">
-            <!-- Breadcrumbs-->
-            <ol class="breadcrumb">
-              <li class="breadcrumb-item"><a href="#">Dashboard</a></li>
-              <li class="breadcrumb-item active">Instructors</li>
-            </ol>
-  
-            <!-- dashboard card -->
-            <div class="card mb-3">
-              <div class="card-header">
-                <i class="fas fa-fw fa-user"></i> Instructor
-                <i class="fas fa-sync fa-spin" id="refreshButton"></i>
-              </div>
-              <div class="card-body">
-                <div
-                  class="alert alert-danger"
-                  id="instructorErrorAlert"
-                  style="display: none"
-                ></div>
-                <table
-                  class="table table-striped"
-                  id="instructorsTable"
-                  style="min-width: 100%"
-                >
-                  <thead>
-                    <tr>
-                      <th>Instructor's Name</th>
-                      <th>Instructor's Bio</th>
-                      <th>Instructor's Role</th>
-                      <th>Instructor's TwitterId</th>
-                      <th>Instructor's GithubId</th>
-                      <th>Actions</th>
-                    </tr>
-                  </thead>
-                  <tbody></tbody>
-                </table>
-              </div>
-              <div class="card-footer text-right">
-                <button class="btn btn-primary" id="addButton" disabled>
-                  Add Instructor
-                </button>
-              </div>
-            </div>
-            <!-- Sticky Footer -->
-            <footer class="sticky-footer">
-              <div class="container my-auto">
-                <div class="copyright text-center my-auto">
-                  <span>Copyright © DSC FUTA 2018</span>
+          <!-- card-grid -->
+          <div class="card-grid">
+            <h1 class="grid-heading mdc-typography--headline3">
+                Instructors</h1>
```

Comment:
For uniformity among the pages, this can be removed.

Reply:
done